llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -8738,10 +8736,9 @@ addUsersInExitBlock(VPlan &Plan,
     if (V->isLiveIn())
       continue;
 
-    VPValue *Ext = B.createNaryOp(
-        VPInstruction::ExtractFromEnd,
-        {V, Plan.getOrAddLiveIn(ConstantInt::get(
-                IntegerType::get(ExitBB->getContext(), 32), 1))});
+    VPValue *Ext = B.createNaryOp(VPInstruction::ExtractFromEnd,
```

Comment:
Might be simpler to retrieve the context from the underlying instruction from `ExitIRI` instead of passing ti explicitly?